### DISCUSSION TITLE
server oracle linux 8.x and ltsp

### DISCUSSION THREAD

**Discussion by sk0al** [ltsp/ltsp#764] [General]
Will the latest ltsp work or run on a oracle linux 8 server (rhel) vm?  Are there any docs or has anyone had success with this?  thank you!

**Comment by alkisg** [comment]
So far a few people requested support for rpm distributions but there wasn't any code or other contribution, so it won't work:
https://github.com/ltsp/ltsp/issues/155

**Reply by sk0al** [reply]
ok thank you.  sounds like something similar will have to be setup and tested by pulling together existing rhel tools, rpms, scripts, etc.
